projects/plugins/jetpack/modules/stats/class-jetpack-stats-upgrade-nudges.php

```diff
@@ -187,26 +226,36 @@ private static function get_upgrade_link( $source ) {
 	 * Outputs one Upgrade item
 	 *
 	 * @param string  $title The title of the item.
+	 * @param string  $text The description of the item.
 	 * @param string  $icon The path of the icon, relative to Jetpack images folder.
 	 * @param string  $link The link of the button.
 	 * @param string  $tracks_id The id used to identify the tracks event. Automatically prefixed with "jetpack_stats_nudges_".
+	 * @param string  $learn_more_link The target of the "Lear more" link.
 	 * @param boolean $subitem Whether it is a subitem or not.
 	 * @param string  $button_label The button label.
 	 * @return void
 	 */
-	private static function print_item( $title, $icon, $link, $tracks_id, $subitem = false, $button_label = null ) {
+	private static function print_item( $title, $text, $icon, $link, $tracks_id, $learn_more_link, $subitem = false, $button_label = null ) {
 		$additional_classes = $subitem ? 'jp-stats-report-upgrade-subitem' : '';
 		$button_class       = $subitem ? 'is-secondary' : 'is-primary';
 		$icon_url           = plugins_url( '', JETPACK__PLUGIN_FILE ) . '/images/' . $icon;
 		$button_label       = is_null( $button_label ) ? __( 'Upgrade', 'jetpack' ) : $button_label;
 		?>
-			<div class="dops-card dops-banner has-call-to-action is-product <?php echo esc_attr( $additional_classes ); ?>">
+			<div class="dops-card dops-banner has-call-to-action is-product jp-stats-report-upgrade-item <?php echo esc_attr( $additional_classes ); ?>">
 				<div class="dops-banner__icon-plan">
 					<img src="<?php echo esc_attr( $icon_url ); ?>" alt="" width="32" height="32">
 				</div>
 				<div class="dops-banner__content">
 					<div class="dops-banner__info">
-						<div class="dops-banner__title"><?php echo esc_html( $title ); ?></div>
+						<div class="dops-banner__title">
+							<?php echo esc_html( $title ); ?>
+							<p>
+								<?php echo esc_html( $text ); ?>
+								<a href="<?php echo esc_attr( $learn_more_link ); ?>">
```

Comment:
What do you think of opening the 'Learn more' links in a new tab?

Reply:
Also, do you think there would be any value to adding tracks ids to the learn more links?

Reply:
> Also, do you think there would be any value to adding tracks ids to the learn more links?

We can track those clicks via the redirect service and I'm don't see any value in building funnels from them, so in my opinion we don't need it. But I'd ask @robertbpugh

Reply:
> What do you think of opening the 'Learn more' links in a new tab?

Done